---
layout: default
title: About
---

### The Microbial G*enomes* A*nd* M*etagenomes* *to* U*nravel* T*raits* (mGAMUT) Lab

--------------------------------------------------------------------------------

**Our lab at Stony Brook University is now open and is recruiting at all levels. See our [join us](/join.html) page for details!**. 

--------------------------------------------------------------------------------

Our research leverages environmental genomic data ("metagenomes") to untangle the dense networks of interactions by which microbes produce community-level outputs relevant to ecosystem health. Microbes play a critical role in regulating global biogeochemical cycles (i.e., how biologically-relevant elements like carbon move through living and non-living systems) – and a deep understanding of microbial growth at the community scale is essential in order to build accurate models to predict future conditions and design appropriate global mitigation strategies in the face of rapid climate change. While microbial ‘omic datasets are increasingly rich in detail, we lack the tools to link these fine-scale descriptions of community composition to community behavior and outputs. We develop open-source, trait-based frameworks that bridge that gap. For example, we maintain an open-source and user-friendly R package, [gRodon](https://github.com/jlw-ecoevo/gRodon2), to estimate the maximum growth rates of mixed-species communities directly from environmental metagenomes.  While our research is primarily computational, we work closely with wet-lab collaborators.

![gRodon](/img/gRodon_concept.png)

More broadly, we develop new tools to infer complex microbial traits from environmentally-derived DNA and apply these tools to large datasets in order to assess the distribution of traits across species and environments. In doing so, we hope to answer questions about (1) what microbes are doing/can do in different habitats, (2) which microbes in a habitat perform particular functions, (3) how certain traits may lead to a fitness benefit/detriment in a particular environmental context, and (4) how complex traits evolve across the tree of life.

In order to accomplish these goals, in addition to developing **novel bioinformatics approaches** and **predictive models** to analyze genomic and metagenomic data, we develop **dynamical models** to construct precise hypotheses about trait evolution and collaborate with experimentalists and field biologists to test these hypotheses.

![Maximum Growth Rate Marine](/img/BIOGEOTRACES_panels.png)
