---
layout: default
title: About
---

### The Microbial G*enomes* A*nd* M*etagenomes* *to* U*nravel* T*raits* (mGAMUT) Lab

--------------------------------------------------------------------------------

**Our lab at Stony Brook University is now open and is recruiting at all levels. See our [join us](/join.html) page for details!**. 

--------------------------------------------------------------------------------

Microbes live and grow in diverse and complex communities, and through dense networks of interactions as a community produce system-level outputs relevant to human (*e.g.*, through control of metabolites in the human gut) and ecosystem (*e.g.*, through control of global carbon cycles) health. These interactions occur on the basis of the traits or “functions” particular to each organism. Yet, our ability to survey complex microbial traits in situ for natural communities is limited at best, and is largely restricted to analyzing the presence or absence of particular metabolic pathways across metagenomes. Microbes, though, are more than what they eat. 

Our work closes the gap between genomic information and emergent community properties via inference of complex traits and life-history strategies at the community level. We map metagenome to (meta)phenotype through the development of novel bioinformatic and mathematical modeling approaches. We then link these (meta)phenotypes to community dynamics and outputs by developing models that bridge cellular, community, and landscape scales. Read more about our ongoing work on our [projects](/projects.html) page.

![mGAMUT](/img/mgamut_green_background_round.png)