---
layout: default
title: People
---


# Principal Investigator

## JL Weissman (they/she)

*Assistant Professor, Department of Ecology & Evolution and the Institute for Advanced Computational Science (IACS)*

<img align="left" src="/img/headshot1c.png" width="200px" style="padding-right: 15px">

**Email:** <jackie.weissman@stonybrook.edu>

**Bluesky:** [@jlw-ecoevo.bsky.social](https://bsky.app/profile/jlw-ecoevo.bsky.social)

**[Google Scholar](https://scholar.google.com/citations?user=IaAUSiQAAAAJ&hl=en)**

**[CV](https://jlw-ecoevo.github.io/CV_JLW.pdf)**

**Bio:**  Jackie Lee “JL” Weissman (they/she) is an Assistant Professor at Stony Brook University in the Department of Ecology and Evolution and at the Institute for Advanced Computational Science.

<details>

<summary>Click for more info</summary>

Her research examines how microbes survive and thrive across diverse environments. She develops new tools to infer what microbes are doing and can do from DNA sequences captured directly from the environment (“metagenomes”), aiming to improve the representation of microbially-mediated biogeochemical cycles in global climate models. She also has a special interest in using a combination of comparative genomics, population genetics, and mathematical models to understand the ancient and ongoing battle between microbes and their viruses. She believes all students, with supportive training and mentorship, can become highly-capable computational biologists, and loves to show students how a little coding can go a long way.

Previously, JL served as the inaugural Director for Proposal Development at the City College of New York, where they managed large, interdisciplinary efforts to bring center-level funding to the college and trained early-career researchers in grantmaking. They maintain research affiliations in biology at CCNY and the University of Southern California and have taught at The Cooper Union School of Art. Before returning to New York, they were faculty at Chapman University, where they ran a computational biology research lab, taught, and developed initiatives to improve mentorship at the college level. 

</details>

# Postdoctoral Researchers

## Alexandra Walling (they/them)

<img align="left" src="/img/walling.png" width="200px" style="padding-right: 15px">

**Email:** <alexandra.walling@stonybrook.edu>

**[CV](https://jlw-ecoevo.github.io/cv_academia_walling-jan-2025.pdf)**

**Bio:** Alexandra (they/them) is a postdoctoral researcher in the mGAMUT ab.

<details>

<summary>Click for more info</summary>

They earned their B.S. in Biology at California State University, Monterey Bay before joining the American Museum of Natural History first as a Helen Fellow and then as a PhD student through the Richard Gilder Graduate School. Their research at AMNH has included comparative questions in prokaryotic and viral evolution. Their PhD work used comparative genomic and phylogenetic techniques to untangle how horizontal gene transfer and secondary loss shaped the evolution of photosynthesis in Erythrobacteraceae, a family of Alphaproteobacteria which show a scattered distribution of a gene cluster encoding photosynthesis. Alexandra's first postdoc at the University of Rhode Island was spent focusing on the problem of how we can filter large genomic datasets to increase signal and reduce noise by developing tools to predict which genomic loci have the most utility for downstream phylogenetic analysis. They used extensive simulations and compared machine learning models for phylogenetic utility determination. 

Alexandra is broadly interested in using a wide range of computational tools to answer questions related to microbial evolution and ecology, particularly how horizontal gene transfer among populations of bacteria may drive bacterial evolution in response to biogeochemical cycling. Alexandra also enjoys teaching and mentoring undergraduate and graduate students just starting out in evolutionary and computational biology. When not working, Alexandra enjoys exploring the great outdoors with their former laboratory beagle, Darwin.

</details>

# Graduate Students

See the [join us](/join.html) page for details.

# Undergraduates

See the [join us](/join.html) page for details.

# Past Mentees

## Stony Brook University

### Jesse Natarajan (she/her)

<img align="left" src="/img/jesse.png" width="200px" style="padding-right: 15px">

**Bio:** Jesse graduated with a Computer Science major and Biology/Chemistry minor from Stony Brook University. She completed her undergraduate honors thesis in the mGAMUT lab on predicting microbial growth rates using short genomic sequences.

<br clear="left"/>


## Chapman University

- Athenna Gonzalez (Spring 2023)

## University of Southern California

- Edward-Robert Dimbo (Spring 2021 - Present; [Genomics and Geobiology Undergraduate Research Experience](https://www.darkenergybiosphere.org/education-diversity/for-undergraduates/ggure/), Senior Thesis, Postbac; coauthor on Weissman et al. 2022, bioRxiv and Dimbo et al. *in prep*)
- James Rosas  (Summer 2021, [National Summer Undergraduate Research Project](https://nsurp.org/))
- Yuniba Yagües (Summer 2020, [National Summer Undergraduate Research Project](https://nsurp.org/), coauthor on Weissman et al. 2022, bioRxiv)
- Oscar Escobedo (Summer 2020, [National Summer Undergraduate Research Project](https://nsurp.org/))

## University of Maryland College Park

- Rohan Laljani (Fall 2017 - Summer 2019; co-author on Weissman et al. 2019, *ISME J* and poster at UMD Biosciences Research Day)
- Vinay Veluvolu (Summer 2018 - Summer 2019; poster at UMD Biosciences Research Day)
- Julia Gall (Fall 2018 - Spring 2019; College Park Scholars project & poster presentation)
- Cori Butkiewicz (Summer 2017 - Spring 2018; now in grad. program at UW-Madison)
- Nicholas Penn (Spring 2017 - Summer 2017)
